driver/dm_reducecalculation.cpp

```diff
@@ -34,6 +34,11 @@ static Driver* makeDriver(const std::string& base_arg)
         return new ReduceCalculationDriver<float16, float>();
     if(base_arg == "reducecalculationbfp16")
         return new ReduceCalculationDriver<bfloat16, float>();
+    if(base_arg == "reducecalculationuint8")
+        return new ReduceCalculationDriver<uint8_t>();
```

Comment:
actually, if input `dtype=uint8`, it will be automatically convert to `int8` (i.e., `dtype=MIOpenInt8`) in `MIOpenDriver`, so im not sure if we need to specify both `int8`, `uint8` or not

Reply:
in case of this, just specify only `uint8`. When it is converted to `dtype=MIOpenInt8`, you just need to cast it to proper type in kernel (`dtype=MIOpenInt8` just mean transferring 8 bits around)

Reply:
so for now, lets just separate and specify both int8_t and uint8_t